src/main/java/core/basesyntax/service/imp/ReaderServiceImp.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.imp;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImp implements ReaderService {
+    @Override
+    public List<String> readRecords(Path path) {
+        try {
+            return Files.readAllLines(path);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t open file " + path);

```

Comment:
path.getFileName() (if we talking about file)

Reply:
It means I want to write the name and local path of the file. 
I think it improves the informativity of the message.
next the same